apps/ui/src/components/App/Nav.vue

```diff
@@ -42,15 +43,17 @@ const space = computed(() =>
     : null
 );
 
-const isController = computedAsync(async () => {
-  if (!networkId.value || !space.value) return false;
-
-  const { account } = web3.value;
+const controller = computedAsync(async () => {
+  if (!networkId.value || !space.value) return null;
 
   const network = getNetwork(networkId.value);
-  const controller = await network.helpers.getSpaceController(space.value);
+  return await network.helpers.getSpaceController(space.value);
+});
 
-  return compareAddresses(controller, account);
+const isController = computed(() => {
+  const { account } = web3.value;
+
+  return compareAddresses(controller.value ?? '', account);
```

Comment:
nit. I'd just check if it's undefined as separate case to avoid calling `compareAddresses` with fallback value.